Source/Mosa.Demo.SVGAWorld.x86/Plugs/FilePlug.cs

```diff
@@ -1,30 +1,29 @@
-﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
+// Copyright (c) MOSA Project. Licensed under the New BSD License.
 
 using System;
 using System.Collections.Generic;
 using Mosa.FileSystem.FAT;
+using Mosa.Runtime.Plug;
 
-namespace Mosa.Demo.SVGAWorld.x86.Utils;
+namespace Mosa.Demo.SVGAWorld.x86.Plugs;
 
-public static class FileManager
+public static class FilePlug
```

Comment:
plugs and their implementations should be as minimal as possible, this should be split out into two classes `FileManager` and `FilePlug`